test/testdata/goldens/outdated/outdated_test/circular dependency on root.txt

```diff
@@ -124,6 +125,7 @@ $ pub outdated --json --no-dev-dependencies
       "package": "foo",
       "kind": "direct",
       "isDiscontinued": false,
+      "isRetracted": false,
```

Comment:
This looks a bit wrong - should it not be the versions knowing if they are retreacted?

Reply:
Renamed this is much better

Reply:
Renamed this to isCurrentRetracted to clarify.